src/EFCore/ModelBuilder.cs

```diff
@@ -168,6 +199,34 @@ public virtual ModelBuilder Entity<TEntity>([NotNull] Action<EntityTypeBuilder<T
             return this;
         }
 
+        /// <summary>
+        ///     <para>
+        ///         Performs configuration of a given entity type in the model. If the entity type is not already part
+        ///         of the model, it will be added to the model.
+        ///     </para>
+        ///     <para>
+        ///         This overload allows configuration of the entity type to be done in line in the method call rather
```

Comment:
Very small suggestion. I've always seen "inline" as one word.
```suggestion
        ///         This overload allows configuration of the entity type to be done inline in the method call rather
```